supported-standards/lens-open-algorithm-standards/models/publication/feed/global-schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://lens.xyz/post.feed.schema.json",
```

Comment:
this doesnt exist?

Reply:
I added the $id so that the schema can be referenced in OpenAPI specs and other JSON schemas. Note: per the JSON schema [spec](https://json-schema.org/draft/2020-12/json-schema-core.html#name-the-id-keyword:~:text=an%20identifier%20and-,not%20necessarily%20a%20network%20locator,-.%20In%20the%20case) $id does not have to be a network locator. I can change the URI to be publication instead of post in accordance with the previous comments.